File: app/src/main/java/com/lifedawn/bestweather/commons/enums/Flickr.java
package com.lifedawn.bestweather.commons.enums;

public class Flickr {
	public enum Time {
		sunrise("sunrise"), sunset("sunrise"), day("day"), night("night");

		private final String text;

		Time(String text) {
			this.text = text;
		}

		public String getText() {
			return text;
		}
	}

	public enum Weather {
		clear("clear"), partlyCloudy("partly_cloudy"), mostlyCloudy("mostly_cloudy"), overcast("overcast"), rain("rain"), snow("snow");

		private final String text;

		Weather(String text) {
			this.text = text;
		}

		public String getText() {
			return text;
		}
	}
}

File: app/src/main/java/com/lifedawn/bestweather/commons/constants/Flickr.kt
package com.lifedawn.bestweather.commons.constants

class Flickr {
    enum class Time(val text: String) {
        sunrise("sunrise"), sunset("sunrise"), day("day"), night("night");

    }

    enum class Weather(val text: String) {
        clear("clear"), partlyCloudy("partly_cloudy"), mostlyCloudy("mostly_cloudy"), overcast("overcast"), rain("rain"), snow("snow");

    }
}